File: .java
public class Main {
    public static void main(String[] args) {
        int total = 0;
        for (int i = 1; i <= 5; i++) {
            total += i;
            System.out.println("Menambahkan " + i + ", total: " + total);
        }
        System.out.println("Hasil akhir: " + total);
    }
}

File: .go
package main
import "fmt"

func main() {
	total := 0
	for i := 1; i <= 5; i++ {
		total += i
		fmt.Printf("Menambahkan %d, total: %d\n", i, total)
	}
	fmt.Printf("Hasil akhir: %d\n", total)
}
